modules/ztp-checking-du-cluster-config.adoc

```diff
@@ -0,0 +1,518 @@
++
+[source,terminal]
+----
+$ oc get namespaces -A -o jsonpath='{range .items[*]}{.metadata.name}{" -- "}{.metadata.annotations.workload\.openshift\.io/allowed}{"\n"}{end}'
+----
++
+.Example output
+[source,terminal]
+----
+default --
+openshift-apiserver -- management
+openshift-apiserver-operator -- management
+openshift-authentication -- management
+openshift-authentication-operator -- management
+----
++
+[IMPORTANT]
+====
+Additional Operators must not be annotated for workload partitioning. In the output from the previous command, additional Operators should be listed without any value on the right-hand side of the `--` separator.
+====
+
+. Check that the `ClusterLogging` configuration is correct.
+
+.. Validate that the curation schedule is appropriate for your application by running the following command:
++
+[source,terminal]
+----
+$ oc get -n openshift-logging ClusterLogging
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  managementState: "Managed"
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+  collection:
+    logs:
+      type: "fluentd"
+      fluentd: {}
+----
+
+.. Validate that the appropriate input logs and destination outputs are configured:
++
+[source,terminal]
+----
+$ oc get -n openshift-logging ClusterLogForwarder
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+    - type: "kafka"
+      name: kafka-open
+      url: tcp://10.46.55.190:9092/test
+  inputs:
+  - name: infra-logs
+    infrastructure: {}
+  pipelines:
+    - name: audit-logs
+      inputRefs:
+       - audit
+      outputRefs:
+       - kafka-open
+    - name: infrastructure-logs
+      inputRefs:
+       - infrastructure
+      outputRefs:
+       - kafka-open
+----
+
+. Check that the web console is disabled (`managementState: Removed`):
++
+[source,terminal]
+----
+$ oc get consoles.operator.openshift.io cluster -o jsonpath="{ .spec.managementState }"
+----
++
+.Example output
+[source,terminal]
+----
+Removed
+----
+
+. Check that `chronyd` is disabled on the cluster node by running the following commands:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
++
+Check the status of `chronyd` on the node:
++
+[source,terminal]
+----
+sh-4.4# chroot /host
+----
++
+[source,terminal]
+----
+sh-4.4# systemctl status chronyd
+----
++
+.Example output
+[source,terminal]
+----
+● chronyd.service - NTP client/server
+   Loaded: loaded (/usr/lib/systemd/system/chronyd.service; disabled; vendor preset: enabled)
```

Comment:
We see following on 4.10 clusters with recommended configs. Note that service is enabled. @imiller0 is this expected for 4.10?

● chronyd.service - NTP client/server
   Loaded: loaded (/usr/lib/systemd/system/chronyd.service; enabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/chronyd.service.d
           └─20-conditional-start.conf
   Active: inactive (dead) since Mon 2022-07-18 10:55:01 UTC; 3h 33min ago

Reply:
Updated based on Yang's output. @imiller0 please confirm.

Reply:
The chronyd fix has not yet been backported to 4.10. When it is we will need to come back and fix this, but for 4.10 yes this is accurate (it is loaded but inactive).